setup.py

```diff
@@ -32,6 +30,6 @@
      , version = version
      , zip_safe = False
      , package_data = {'aspen': ['request_processor/mime.types']}
-     , install_requires = ASPEN_DEPS
-     , tests_require = TEST_DEPS
+     , install_requires = open('requirements.txt').read()
+     , tests_require = open('requirements_tests.txt').read()
```

Comment:
No `spitlines`, it would probably fail on empty lines and comments if we naively split the lines.